src/app/pages/TopicPage/Curation/CurationPromo/index.jsx

```diff
@@ -14,20 +20,55 @@ const CurationPromo = ({
   duration: mediaDuration,
   headingLevel,
 }) => {
+  const { translations } = useContext(ServiceContext);
+
+  const audioTranslation = path(['media', 'audio'], translations);
+  const videoTranslation = path(['media', 'video'], translations);
+  const photoGalleryTranslation = path(['media', 'photogallery'], translations);
+  const durationTranslation = path(['media', 'duration'], translations);
+  const duration = moment.duration(mediaDuration, 'seconds');
+
+  const separator = ',';
+
+  const formattedDuration = formatDuration({ duration, separator });
+  const durationString = `${durationTranslation}, ${formattedDuration}`;
+
   return (
     <Promo>
       <Promo.Image src={imageUrl} alt={imageAlt} lazyLoad={lazy}>
-        <Promo.MediaIcon type={type}>{mediaDuration}</Promo.MediaIcon>
+        <Promo.MediaIcon type={type}>
+          {mediaDuration &&
+          (type === 'video' || type === 'audio' || mediaDuration > 0)
```

Comment:
This could be extracted out into a variable to make it a bit easier to read, e.g:
`const showDuration = mediaDuration && ['video', 'audio'].includes(type)`

I think checking for just `mediaDuration` is the same as checking `mediaDuration > 0`, so that last check may not be needed.